convert2rhel/systeminfo.py

```diff
@@ -115,9 +123,8 @@ def _get_cfg_section(self, section_name):
                                  " conversion to RHEL.")
 
         options_list = cfg_parser.options(section_name)
-        return dict(zip(options_list,
-                        map(lambda opt: cfg_parser.get(section_name, opt),
-                            options_list)))
+        return dict(list(zip(options_list,
```

Comment:
It should work without list, i.e. dict(zip...

Reply:
Good catch, most of the code changes are from a python 2/3 compatibility tool. I looked it up though and in Py2 zip returns a list, whilst in Py3 it returns an iterator

I can't check whether it works since the tests for Py3 doesn't work yet, but I'll leave it as `list(zip())`